components/11-argo-workflow/base/templates/cluster.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
```

Comment:
This whole config file assumes that you're running on kind and will fail in any other cluster.

Reply:
hmm.. so I am thinking overlay w/ dev vs prod.  Would that work?  This can remain a template